docs/usage/tracing.asciidoc

```diff
@@ -0,0 +1,72 @@
+[[ecs-tracing-usage]]
+==== Tracing Usage
+
+A trace describes the individual operations, and their causality, that ensue as a result of a single logical 
+operation. A distributed trace comprises operations across multiple distributed components, crossing process, 
+network, and security boundaries.
+
+For example, a distributed trace may be initiated when someone presses a button to start an action on a website - in this example, 
+the trace will represent calls made between the downstream services that handled the chain of requests initiated by this 
+button being pressed.
+
+A trace is made up of transactions and spans, where each transaction/span is one operation, activity, or part of the overall
+trace. A transaction is a special kind of span that is the highest level of work measured within a service. In Elastic APM, 
+transactions have additional attributes associated with them 
+(for more detail see https://www.elastic.co/guide/en/apm/get-started/current/transactions.html).
+
+Usually a transaction corresponds to the top level request to a component. Sometimes a component may break up processing into 
+multiple spans (and transactions) if needed. 
+
+The spans (and transactions) have a parent/child relationship with other spans and form a tree (or directed acylic grapph), 
+with each span (or transaction), except the root, having a parent.
+
+The `trace.id` allows the distributed trace messages to be correlated across an entire logical operation.
+
+The `span.id` allows messages from a single section of processing within the overall trace, e.g. a request to a component, to be correlated together. The `transaction.id` is the equivalent for transaction spans.
+
+If following the https://www.w3.org/TR/trace-context/[W3C Trace Context Recommendation] then
+the values should correspond to those specified in the recommendation and passed in the `traceparent` header, 
+with `trace.id` being 32 lowercase hex characters, and `span.id` being 16 lowercase hex characters.
+
+For correlation wtih Elastic Application Performance Monitoring (APM) see 
+the https://www.elastic.co/guide/en/apm/get-started/current/distributed-tracing.html[APM Distributed Tracing] documentation.
```

Comment:
We've already used APM on this page, so I think only using the initialism is acceptable.

```suggestion
For correlation with Elastic APM, see
{apm-overview-ref-v}/distributed-tracing.html[APM Distributed Tracing].
```